attributes/default.rb

```diff
@@ -33,6 +33,8 @@
 default['onlinefs']['monitoring']             = 12800
 
 default['onlinefs']['service']['thread_number'] = 10
+default['onlinefs']['service']['ron_db_thread_number'] = 10
+default['onlinefs']['service']['vector_db_thread_number'] = 5
```

Comment:
5 for `vector_db_thread_number`?

Reply:
tbh it is arbitrary, but I expect there should be much embedding fg than normal fg, which means much less write to vector db than to rondb.